Client/Frontend/Strings.swift

```diff
@@ -3747,9 +3747,9 @@ extension String {
             value: "Adjusted rating",
             comment: "Title of the adjusted rating card displayed in the shopping review quality bottom sheet.")
         public static let AdjustedRatingDescription = MZLocalizedString(
-            key: "Shopping.AdjustedRating.Description.v120",
```

Comment:
Like this all strings will be deleted for v120 if this is exported to l10n. Are we ok with this? Referring to explanation in [wiki](https://github.com/mozilla-mobile/firefox-ios/wiki/How-to-add-and-modify-Strings#how-to-modify-a-string)